Handler/Wiki.hs

```diff
@@ -494,6 +494,7 @@ getWikiEditR project_handle language target wiki_edit_id = do
 
 getNewWikiR :: Text -> Language -> Text -> Handler Html
 getNewWikiR project_handle language target = do
+    runYDB (coPageInfo project_handle language target)
```

Comment:
I thought maybe, but then shouldn't it be a function that takes some page info and gives you the project, language, and target? :P

Also, I made the original comment before waking up.

Anyway, I'll merge this tomorrow am, though I might change the name of this func to something like.. requireNonexistent (to follow the 'requireAuth' lead).